full/Dockerfile

```diff
@@ -30,21 +30,34 @@ WORKDIR $HOME
 # custom Bash prompt
 RUN { echo && echo "PS1='\[\e]0;\u \w\a\]\[\033[01;32m\]\u\[\033[00m\] \[\033[01;34m\]\w\[\033[00m\] \\\$ '" ; } >> .bashrc
 
+### Gitpod user (2) ###
+USER gitpod
+# use sudo so that user does not get sudo usage info on (the first) login
+RUN sudo echo "Running 'sudo' for Gitpod: success"
+# create .profile.d folder and source it in the bashrc
+RUN mkdir /home/gitpod/.profile.d && \
+    echo "for i in \$(ls \$HOME/.profile.d/*); do source \$i; done" >> /home/gitpod/.profile
+
 ### Install C/C++ compiler and associated tools ###
+LABEL dazzle/layer=lang-c
+LABEL dazzle/test=tests/lang-c.yaml
+USER root
 RUN curl -fsSL https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - \
     && apt-add-repository -yu "deb http://apt.llvm.org/disco/ llvm-toolchain-disco main" \
     && apt-get install -yq \
         clang-format \
         clang-tidy \
-        clang-tools \
+        # clang-tools \ # breaks the build atm
```

Comment:
Is this a test leftover?

Reply:
No. There's an issue with the clang apt sources/packages at the moment. This is independent of this PR/dazzle. The package itself just doesn't install at the moment.

```
Unpacking clangd (1:10.0-48+0~20191017130034.13~1.gbp24c8a8) ...
dpkg: error processing archive /tmp/apt-dpkg-install-kqkfT1/26-clangd_1%3a10.0-48+0~20191017130034.13~1.gbp24c8a8_amd64.deb (--unpack):
 trying to overwrite '/usr/bin/clangd', which is also in package clang-tools 1:8.0-48~exp1ubuntu1
```